src/engraving/libmscore/bracketItem.cpp

```diff
@@ -51,7 +51,7 @@ mu::engraving::PropertyValue BracketItem::getProperty(Pid id) const
     case Pid::BRACKET_COLUMN:
         return _column;
     case Pid::BRACKET_SPAN:
-        return _bracketSpan;
+        return int(_bracketSpan);
```

Comment:
Hmm, not `static_cast<int>(_bracketSpan)`?

Reply:
fixed